app/models/event.rb

```diff
@@ -0,0 +1,4 @@
+class Event < ApplicationRecord
+  belongs_to :user
+  has_many :attendees
```

Comment:
The event should have a creator ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/user_assoc_requirement.png) from Odin)

Remember to include the class_name in your associations.

Reply:
We Add the forign key of the user who created the event with 'user_id' and you can see in the schema.rb
![image](https://user-images.githubusercontent.com/12748812/81272472-f8980e00-9066-11ea-97a2-8d179b6e7512.png)